pytest_snapshot/_utils.py

```diff
@@ -44,7 +45,7 @@ def might_be_valid_filename(s: str) -> bool:
     )
 
 
-def simple_version_parse(version: str):
+def simple_version_parse(version: str) -> Tuple[int, int, int]:
```

Comment:
Either leave this as `Tuple[int, ...]`, or add a `cast(..., <expr>)` to the last line. Otherwise mypy will complain.